bin/varnishd/cache/cache.h

```diff
@@ -287,6 +287,8 @@ struct boc {
 	enum boc_state_e	state;
 	uint8_t			*vary;
 	uint64_t		len_so_far;
+	uint64_t		client_data_so_far;
```

Comment:
I would rename `len_so_far` to `fetched_so_far` and `client_data_so_far` to `delivered_so_far` for clarity.